.github/workflows/benchmark.yml

```diff
@@ -0,0 +1,25 @@
+name: Run benchmarks
+
+on:
+  pull_request:
+
+jobs:
+  Benchmark:
+    runs-on: ubuntu-latest
+    env:
+      JULIA_DEBUG: BenchmarkCI
+    steps:
+      - uses: actions/checkout@v2
+      - uses: julia-actions/setup-julia@latest
+        with:
+          version: 1.3
+      - name: Install dependencies
+        run: julia -e 'using Pkg; pkg"add PkgBenchmark https://github.com/tkf/BenchmarkCI.jl"'
+      - name: Run benchmarks
+        run: julia -e 'using BenchmarkCI; BenchmarkCI.judge(baseline = "HEAD")'
```

Comment:
```suggestion
        run: julia -e 'using BenchmarkCI; BenchmarkCI.judge()'
```

Reply:
Do we need to commit this? @tkf

Reply:
Yes, it was for build passing even when origin/master (default `baseline`) is not ready for benchmark.